[2024-07-17T00:47:54.151+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-17T00:47:54.167+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_ticker.etl_usd_mxn scheduled__2024-07-14T17:50:00+00:00 [queued]>
[2024-07-17T00:47:54.171+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_ticker.etl_usd_mxn scheduled__2024-07-14T17:50:00+00:00 [queued]>
[2024-07-17T00:47:54.172+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-07-17T00:47:54.180+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): etl_usd_mxn> on 2024-07-14 17:50:00+00:00
[2024-07-17T00:47:54.183+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=28182) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-17T00:47:54.184+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_ticker', 'etl_usd_mxn', 'scheduled__2024-07-14T17:50:00+00:00', '--job-id', '368', '--raw', '--subdir', 'DAGS_FOLDER/bid_ask_dag.py', '--cfg-path', '/tmp/tmpb13rash3']
[2024-07-17T00:47:54.185+0000] {standard_task_runner.py:63} INFO - Started process 28184 to run task
[2024-07-17T00:47:54.185+0000] {standard_task_runner.py:91} INFO - Job 368: Subtask etl_usd_mxn
[2024-07-17T00:47:54.210+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_ticker.etl_usd_mxn scheduled__2024-07-14T17:50:00+00:00 [running]> on host 6f712de23803
[2024-07-17T00:47:54.253+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_ticker' AIRFLOW_CTX_TASK_ID='etl_usd_mxn' AIRFLOW_CTX_EXECUTION_DATE='2024-07-14T17:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-14T17:50:00+00:00'
[2024-07-17T00:47:54.254+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-17T00:47:54.262+0000] {logging_mixin.py:188} WARNING -   0%|          | 0/600 [00:00<?, ?it/s]
[2024-07-17T00:47:55.393+0000] {bid_ask_dag.py:70} INFO - Threshold! {'orderbook_timestamp': '2024-07-17T00:47:53+00:00', 'book': 'usd_mxn', 'bid': 17.63, 'ask': 17.65, 'spread': 0.11331444759206558}
[2024-07-17T00:47:55.395+0000] {logging_mixin.py:188} WARNING -   0%|          | 0/600 [00:01<?, ?it/s]
[2024-07-17T00:47:55.396+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-17T00:47:55.396+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/bid_ask_dag.py", line 71, in etl_spread
    SlackWebhookHook(
    ^^^^^^^^^^^^^^^^
NameError: name 'SlackWebhookHook' is not defined
[2024-07-17T00:47:55.411+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=etl_ticker, task_id=etl_usd_mxn, run_id=scheduled__2024-07-14T17:50:00+00:00, execution_date=20240714T175000, start_date=20240717T004754, end_date=20240717T004755
[2024-07-17T00:47:55.420+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 368 for task etl_usd_mxn (name 'SlackWebhookHook' is not defined; 28184)
[2024-07-17T00:47:55.456+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-07-17T00:47:55.467+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-17T00:47:55.468+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
